Install.sbt

```diff
@@ -0,0 +1,28 @@
+import sbt.{Def, Keys}
+import Keys.`package`
+import sbt.complete.Parsers._
+
+lazy val installToNetLogo = Def.inputKey[Unit]("Install To NetLogo (Mac OS X only)")
+
+installToNetLogo := {
+  val version: String = (Space ~> StringBasic).parsed
+  val netLogoDir = file(s"/Applications/NetLogo $version/")
+  if (netLogoDir.exists && netLogoDir.isDirectory) {
+    val createdFile = (Keys.`package` in Compile).value
+    val installationDir = netLogoDir / "extensions" / "xw" / "widgets" / "LSWidgets"
+    if (! installationDir.isDirectory) {
+      installationDir.mkdirs()
+      IO.copy(
+        Seq("miglayout-core", "miglayout-swing").map(
+          p => (baseDirectory.value / s"$p-4.2.jar", installationDir / s"$p-4.2.jar")))
+    }
+    val installationFile = installationDir / createdFile.getName
+    if (installationFile.exists) {
+      IO.delete(installationFile)
+    }
+    IO.copy(Seq((createdFile -> installationFile)))
+    println(s"latest LSWidgets installed to ${installationFile.getPath}")
+  } else {
+    println(s"could not find /Applications/NetLogo $version/")
+  }
+}
```

Comment:
I think I would prefer for this not to be in here. It's not cross-platform. Besides, why not just symlink?